src/crate/crash/tabulate.py

```diff
@@ -538,7 +538,7 @@ def _column_type(strings, has_invisible=True):
     True
 
     """
-    types = [_type(s, has_invisible) for s in strings ]
```

Comment:
strings as name implies that there are strings, which is according to your usage not the case